src/JoomlaBrowser.php

```diff
@@ -899,4 +905,18 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+
+	/**
+	 * Method is to set Wait for page title
+	 *
+	 * @param   string   $title    Page Title text
+	 * @param   integer  $waiting  Waiting time
+	 *
+	 * @return  void
+	 */
+	public function waitForPageTitle($title, $waiting = 60)
```

Comment:
to keep consistency with the Codeception functions I would rename the parameter `$waiting` and use `$timeout` like codeception does in its functions:

![screen shot 2016-05-31 at 12 01 11](https://cloud.githubusercontent.com/assets/1375475/15670788/90758656-2727-11e6-8fc8-5345535567b5.png)